archived/broker/README.md

```diff
@@ -99,7 +99,7 @@ the [docker-compose.yaml](#local-demo).
 
 ### Local Demo
 
-There is a [docker-compose.yaml](docker-compose.yaml) that starts a broker and a connector.
+There is a [docker-compose-broker.yaml](../../docker-compose-broker.yaml) that starts a broker and a connector.
```

Comment:
this file should be obsolete once the docker composes are merged and the local demo guide itself was adjusted

unfortunately we need to do that until we are doing the feature for the MDS, so we can properly sell them that we merged the repos.

Reply:
Merged